src/ui/Base/QueryBuilder.ts

```diff
@@ -508,18 +521,10 @@ export class QueryBuilder {
   }
 
   private validateGroupBy() {
-    if (!this.groupByRequests.length && this.facets.length) {
+    if (this.facetRequests && !this.groupByRequests.length) {
```

Comment:
What does this condition mean for a page using both the old and new facets? i.e. if we have `groupByRequests` and `facetRequests` Or will we be telling clients they cannot mix?